### DISCUSSION TITLE
It is possible to disable the 3d rendering of the building for better performance ?

### DISCUSSION THREAD

**Discussion by paul-bonnel-fr** [maplibre/maplibre-gl-js#1554] [Q&A] [answered]
Good afternoon all,

I'm wondering if it is possible to disable the 3d rendering of the building for better performance ?

**Comment by birkskyum** [comment, marked as answer]
You'll need a style without a [fill-extrusion](https://docs.mapbox.com/mapbox-gl-js/style-spec/layers/#fill-extrusion) for that.
